webpack.config.ts

```diff
@@ -165,9 +166,13 @@ const defaults = {
         {
           from: path.join(__dirname, "src", "frontend", "assets"),
           to: path.join(__dirname, "dist"),
-          // We want html files from HtmlWebpackPlugin, not the original ones
           filter: (resourcePath) => {
-            return !resourcePath.endsWith(".html");
+            return (
+              // we want html files from HtmlWebpackPlugin, not the original ones
+              !resourcePath.endsWith(".html") &&
+              // exclude dotfiles
+              !resourcePath.split("/").pop()?.startsWith(".")
```

Comment:
when would there be dotfiles in the dist?

Reply:
> Don't .DS_Store files show up on git status anyway?

No, we have them explicitly ignored in our `.gitignore`. I think this behavior is consistent with that.